schema/catalogue-configmap.json

```diff
@@ -1,8 +1,26 @@
 {
   "type": "object",
   "properties": {
-    "available": {"type": "boolean"}
+    "available": {"type": "boolean"},
+    "storage": {
+      "oneOf": [{
+        "type": "object",
+        "properties": {
+          "type": {
+            "type": "string",
+            "enum": ["nfs"]
+          },
+          "server": {
+            "type": "string"
+          },
+          "rootDirectory": {
+            "type": "string"
+          }
+        },
+        "additionalProperties": false
```

Comment:
I've tried to always pair up "required" with "additionalProperties", I'd add this to line 20:
        "required": ["type", "server", "rootDirectory"],

Reply:
Good spot! When I was making the changes I didn't add the `required` as the `server` and `rootDirectory` might not be applicable for another storage type, completely forgetting that handling that was the whole point of putting it in a `oneOf` 🤦 

I've made the changes as you suggested!